### DISCUSSION TITLE
`LOBStreamBuf` has overridden `rdbuf` API that causes amibiguity in the compiler

### DISCUSSION THREAD

**Discussion by Smurf-IV** [pocoproject/poco#4303] [General]
V 1.12.5
``` c#
        Poco::Data::BLOB compressedFile = pStatFile->XZCompressed();
        Poco::Data::BLOBInputStream blobStreamBuf(compressedFile);
        blobStreamBuf.rdbuf() <----
```
![image](https://github.com/pocoproject/poco/assets/2418812/8d4b6f15-c309-4a58-92e0-595bfe26f585)

**Comment by aleks-f** [comment]
@Smurf-IV the most effective way to get your request fulfilled is to provide a fix, with a testcase (if applicable) to demonstrate that the fix is effective, and send a pull request

**Comment by Smurf-IV** [comment]
I do not know how to fix this, as it seems it was done for a reason I cannot fathom !

**Comment by aleks-f** [comment]
Well, if you would provide a [SSCCE](http://sscce.org), then perhaps someone would be able to help you understand what is happening. I don't see what you're trying to do and how this is a valid issue, moving it to discussions
